protocol-designer/src/pages/Designer/DeckSetup/DeckSetupToolbox.tsx

```diff
@@ -310,11 +316,11 @@ export function DeckSetupToolbox(
                       createdStackForSlot[createdStackForSlot.length - 1]
                     ]
                   }
-                  lidDisplayName={
+                  lidId={
```

Comment:
To avoid passing undefined
```suggestion
                  {...(createdLidForSlot != null &&
                  createdStackForSlot.includes(createdLidForSlot?.id)
                    ? {}
                    : { lidId: createdLidForSlot?.id })}

```